lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -307,12 +106,12 @@
   delegate_to: localhost
   register: deploy
 
-- name: Clone a VM from Template and customize
+- name: Clone a virtual machine from Template and customize
   vmware_guest:
-    hostname: 192.168.1.209
-    username: administrator@vsphere.local
```

Comment:
Is this form of username (name@vsphere.local) required? If there are requirements for the format of the username, it might be helpful to leave one example where a sample value is spelled out.

Reply:
username@domain is common type of username in vSphere environment and generally assumption that VMware administrator knows about this. I am not sure whether to keep this or replace it with variable.